### PR TITLE
fix(notebooks): relaunch errored notebook

### PR DESCRIPTION
**Pull request by christophe-rannou**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #DTRSD-47559
| License          | BSD 3-Clause

## Description

Allow to start 'failed' and 'error' notebooks

### CODE REVIEW

**Review by radireddy — DISMISSED** [review]

File: packages/manager/modules/pci/src/projects/project/notebooks/Notebook.class.js [hunk 1]
```diff
@@ -111,6 +111,14 @@ export default class Notebook {
     return this.status?.state === NOTEBOOK_STATUS.FAILED;
   }
 
+  isErrored() {
+    return this.status?.state === NOTEBOOK_STATUS.FAILED;
+  }
+
+  isTerminal() {
```
Comment (marie-j) [inline packages/manager/modules/pci/src/projects/project/notebooks/Notebook.class.js:118]: Not really getting the meaning here, `isTerminated` ?

File: packages/manager/modules/pci/src/projects/project/notebooks/dashboard/general-information/general-information.html [hunk 2]
```diff
@@ -121,33 +123,42 @@
...
+                                data-ng-bind="'pci_notebooks_general_information_life_cycle_status_' + $ctrl.notebook.stateInfo.state | translate"
+                            ></span>
+                        </span>
                     </oui-tile-description>
 
-                    <oui-action-menu aria-label="Actions" data-compact data-placement="end">
-                        <oui-action-menu-item data-on-click="$ctrl.onNotebookStartClick($ctrl.notebook.id)"
-                                              data-disabled="!$ctrl.notebook.isStopped()">
+                    <oui-action-menu
+                        aria-label="Actions"
```
Comment (marie-j) [inline packages/manager/modules/pci/src/projects/project/notebooks/dashboard/general-information/general-information.html:133]: Maybe translate the aria-label

File: packages/manager/modules/pci/src/projects/project/notebooks/Notebook.class.js (hunk 1 above)
Comment (christophe-rannou) [reply packages/manager/modules/pci/src/projects/project/notebooks/Notebook.class.js:118]: It means that the notebook is in a terminal state (several of them), `terminated` would be more of a state.

File: packages/manager/modules/pci/src/projects/project/notebooks/dashboard/general-information/general-information.html (hunk 2 above)
Comment (christophe-rannou) [reply packages/manager/modules/pci/src/projects/project/notebooks/dashboard/general-information/general-information.html:133]: Not in the scope of this PR and I do not want to wait for a trad before merging this fix

Comment (marie-j) [reply packages/manager/modules/pci/src/projects/project/notebooks/dashboard/general-information/general-information.html:133]: Ok my bad, I didn't notice it was already here

**Review by antleblanc — CHANGES_REQUESTED** [review]
Good start!

I noticed that some checks were not successful. Could you please have a look?
Thanks!

File: packages/manager/modules/pci/src/projects/project/notebooks/Notebook.class.js [hunk 3]
```diff
@@ -111,6 +111,14 @@ export default class Notebook {
     return this.status?.state === NOTEBOOK_STATUS.FAILED;
   }
 
+  isErrored() {
+    return this.status?.state === NOTEBOOK_STATUS.FAILED;
+  }
```
Comment (JeremyDec) [inline packages/manager/modules/pci/src/projects/project/notebooks/Notebook.class.js:116]:
```suggestion
  isInError() {
    return this.status?.state === NOTEBOOK_STATUS.ERROR;
  }
```

**Review by JeremyDec — CHANGES_REQUESTED** [review]

File: packages/manager/modules/pci/src/projects/project/notebooks/Notebook.class.js (hunk 3 above)
Comment (christophe-rannou) [reply packages/manager/modules/pci/src/projects/project/notebooks/Notebook.class.js]: nice catch thanks

**Review by JeremyDec — APPROVED** [review]

**Review by radireddy — APPROVED** [review]

**Pull request merged**